code/modules/antagonists/brainwashing/brainwashing.dm

```diff
@@ -0,0 +1,57 @@
+/proc/brainwash(mob/living/L, directives)
+	if(!L.mind)
+		return
+	if(!islist(directives))
+		directives = list(directives)
+	var/datum/mind/M = L.mind
+	var/datum/antagonist/brainwashed/B = M.has_antag_datum(/datum/antagonist/brainwashed)
+	if(B)
+		for(var/O in directives)
+			var/datum/objective/brainwashing/objective = new(O)
+			B.objectives += objective
+			M.objectives += objective
+		B.greet()
+	else
+		B = new(directives)
+		M.add_antag_datum(B)
+
+/datum/antagonist/brainwashed
+	name = "Brainwashed Victim"
+	job_rank = ROLE_BRAINWASHED
+	roundend_category = "brainwashed victims"
+	show_in_antagpanel = FALSE
+	antagpanel_category = "Other"
+	show_name_in_check_antagonists = TRUE
+
+/datum/antagonist/brainwashed/New(list/directives)
```

Comment:
I'd prefer if you kept the new function clean for antags, this is not really a must but makes it easier on automated systems if they can assume created antag is correct. Just move it to some separate function called in the helper. You can use the same one to update objectives in there.

Reply:
Alright, will do.